development.cfg

```diff
@@ -1,4 +1,3 @@
 [buildout]
 extends =
-    test-plone-4.2.x.cfg
```

Comment:
We should extend from the 4.3 cfg file instead.

Reply:
👍